### DISCUSSION TITLE
So what next?

### DISCUSSION THREAD

**Discussion by htahir1** [ebhy/budgetml#4] [Ideas]
Wow - The last 24 hours have been absolutely crazy and beyond any of our expectations! Trending on HackerNews and Reddit and more than 500 stars on GitHub is really unexpected and it's awesome to see the simplicity of BudgetML resonated with the community.

The question now is a natural one: What next? And for this I turn to you: Please comment down below how you're using BudgetML, what you think can be improved, and what other features would you like to see. Your feedback is _not going to go to waste_ - on the contrary, it might well shape the future of this community and repository.

Thank you again and wish everyone reading this the best of health

**Comment by BenAsaf** [comment]
I have starred it for now and once I have free-time I'll explore the repository :) hoping to contribute code.
Good job!

**Reply by htahir1** [reply]
Thank you, your support is much appreciated!
